app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java

```diff
@@ -62,7 +62,6 @@ public void detachView() {
 
     public void loadAllAnnouncement(int pageNumber) {
         checkViewAttached();
-        getMvpView().showProgressbar(true);
```

Comment:
why have you removed this line?

Reply:
While announcements are loading, due to this we can see two progress bar at the same time. You can see in this video:- https://drive.google.com/file/d/1GaAdVFM7PGhusLZizwuAo2rIBD2Zj4ly/view?usp=sharing
So, I removed one progress bar in loadAllAnnouncement.

Reply:
Nice!